src/DGEList.py

```diff
@@ -178,42 +217,39 @@ def read_handle(cls, handle: StringIO, **kwargs: Mapping) -> 'DGEList':
 
         return cls(counts=counts, samples=samples, genes=genes)
 
-    def cpm(
-        self,
-        log: bool = False,
-        prior_count: float = PRIOR_COUNT
-    ) -> 'DGEList':
+    def cpm(self, log: bool = False, prior_count: float = PRIOR_COUNT) -> 'DGEList':
         """Return the DGEList normalized to read counts per million."""
-        raise NotImplementedError
         # self.counts = 1e6 * self.counts / np.sum(self.counts, axis=0)
         # if log:
         #     self.counts[self.counts == 0] = prior_count
         #     self.counts = np.log(self.counts)
         # return self
 
+        raise NotImplementedError
+
+        # TODO: Implement here
+
     def rpkm(
-        self,
-        gene_lengths: Mapping,
-        log: bool = False,
-        prior_count: float = PRIOR_COUNT
-    ) -> 'DGEList':
+            self,
+            gene_lengths: Mapping,
+            log: bool = False,
+            prior_count: float = PRIOR_COUNT
+    ) -> None:
         """Return the DGEList normalized to reads per kilobase of gene length
         per million reads.
 
         """
         raise NotImplementedError
 
         # TODO: Implement here
-
         # self = self.cpm(log=log, prior_count=prior_count)
-        return self
 
     def tpm(
-        self,
-        transcripts: Mapping,
-        log: bool = False,
-        prior_count: float = PRIOR_COUNT
-    ) -> 'DGEList':
+            self,
+            transcripts: Mapping,
+            log: bool = False,
+            prior_count: float = PRIOR_COUNT
+    ) -> None:
```

Comment:
Why extra indent?

This does not return `None`. Use `"DGEList"` to mitigate forward referencing:

https://www.python.org/dev/peps/pep-0563/#forward-references

Reply:
It doesn't currently return anything - shouldn't that be changed when the function changes?